app/src/main/java/com/kodeco/android/coordplot/MapComposable.kt

```diff
@@ -23,32 +23,43 @@ fun MapComposable() {
     var xSliderValue by remember { mutableFloatStateOf(0.5f) }
     var ySliderValue by remember { mutableFloatStateOf(0.5f) }
 
-    Column( horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.Center) {
-        MapGraph(xSliderValue, ySliderValue, mapSize = 300, cursorSize = 36)
-        Spacer(modifier = Modifier.size(20.dp))
-        Slider(
-            value = xSliderValue,
-            onValueChange = { newValue ->
-                xSliderValue = newValue
-            },
-            valueRange = 0f..1.0f
-        )
-        Slider(
-            value = ySliderValue,
-            onValueChange = { newValue ->
-                ySliderValue = newValue
-            },
-            valueRange = 0f..1.0f
-        )
-        SliderValues(xSliderValue, ySliderValue)
-    }
-}
 
-@Composable
-fun SliderValues(xSliderValue: Float, ySliderValue: Float) {
-    Text(text = "X Slider Value: $xSliderValue")
-    Text(text = "Y Slider Value: $ySliderValue")
+    BoxWithConstraints {
+        if (maxWidth < maxHeight) {
```

Comment:
It works fine for both portrait and landscape. As an alternate options, you can also use "screenOrientation == Configuration.ORIENTATION_PORTRAIT" or "BoxWithConstraints"